src/components/SeedingRateCard/index.js

```diff
@@ -258,9 +260,7 @@ const SeedingRateChip = ({ value }) => (
       width: '100px',
       padding: '0.5rem',
       margin: '0 auto',
-      backgroundColor: '#E5E7D5',
       border: '#C7C7C7 solid 1px',
```

Comment:
lets make this  2 px and make it a light grey color or add so opacity to make it seem lighter.  the border feels think and too solid of a color compared to the design review sheet